### ISSUE TITLE
Hovers don't work when inside SDK files

### ISSUE THREAD

**Issue by DanTup** [Dart-Code/Dart-Code#158]
```
[16:31:06]: ==> {"id":"135","method":"analysis.getHover","params":{"file":"m:\\Apps\\Dart\\lib\\core\\errors.dart","offset":4171}}
[16:31:06]: <== {"id":"135","result":{"hovers":[]}}
```

**Comment by DanTup** [comment]
Related SDK case: https://github.com/dart-lang/sdk/issues/27241

**Comment by DanTup** [comment]
Fixed by the same fix for #151

**Comment by DanTup** [comment]
Does not appear fixed in 1.20.1.

**Comment by DanTup** [comment]
Works in 1.23 dev.
